### PR TITLE
02.GIT.local

### PR DESCRIPTION
**Pull request by Haroshka**

### CODE REVIEW

File: Ivan-Haroshka/02. GIT. Local/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+02.GIT.local
```
Comment (pluhin) [inline Ivan-Haroshka/02. GIT. Local/README.md:1]: Missing list of commits for each branch

Comment (pluhin) [inline Ivan-Haroshka/02. GIT. Local/README.md:1]: Should be header

**Review by pluhin — CHANGES_REQUESTED** [review]

File: Ivan-Haroshka/02. GIT. Local/README.md (hunk 1 above)
Comment (Haroshka) [reply Ivan-Haroshka/02. GIT. Local/README.md:1]: list added

Comment (Haroshka) [reply Ivan-Haroshka/02. GIT. Local/README.md:1]: header added

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**